docs/courses/design/brainstorming-analogies.md

```diff
@@ -0,0 +1,69 @@
+# Step 1.5 - What analogies or heuristics will you use?
+
+Write a list of analogies for concepts, heuristics for best practices, and any other non-technical explanations of things that may be helpful to students.
+
+
+## Examples
+
+From a course on single cell RNA-seq workflows. This analogy is intuitive even without a biology background, and the origin of the idea is cited.
+
+> To compare bulk and single-cell RNA-seq, I like to use the analogy with the milkshake versus pieces of fruits (idea from Shalek and Regev in [a talk](https://image.slidesharecdn.com/whyscientistanalyzesinglecells-161116191653/95/why-scientist-analyze-single-cells-4-638.jpg?cb=1501073560))
+
+From a course on forecasting product demand. Again, this analogy is likely be intuitive to most people.
+
+> Signal and noise - It's like trying to hear someone across a crowded room. Remove the noise and you can understand easily what they are telling you.
+
+There's a [funny image](https://effectsizefaq.com/2010/05/31/i-always-get-confused-about-type-i-and-ii-errors-can-you-show-me-something-to-help-me-remember-the-difference) analogy used to explain Type I and II statistical errors.
+
+An analogy from Albert Kim:
+
+> R packages are like apps, and CRAN is like the App Store.
+
+[IBM](https://www.ibm.com/support/knowledgecenter/en/SS3RA7_16.0.0/com.ibm.spss.modeler.help/clementine/jython/clementine/svmnodeslots.htm) heuristic on machine learning model choice:
+
+> Support Vector Machines work well with wide data sets, such as those with a very large number of input fields.
+
+Computer science is full of heuristics on good coding practice. From The Elements of Programming Style:
+
+> debugging is twice as hard as writing a program in the first place
+
+From Testing R Code:
+
+> Most R functions should be 7 lines of code or less.
+
+[Siddiqui 2013](https://pdfs.semanticscholar.org/fa7a/1c2e306b2aa2a4e9f8b15de7075246e1e0ba.pdf) provides some heuristics on minimum sample sizes for various statistical techniques
+
+> For multiple regression analyses the desired level is between 15 to 20 observations for each predictor variable.
+
+
+## FAQs
+
+### What is the difference between an analogy and a heuristic?
+
+Analogies are comparisons between similar things. In this case, you want an analogy that compares the technical concept in your course, with a concept from every-day life.
+
+Heuristics are sometimes called "rules of thumb". They are practical rules that work a lot of the time.
+
+### This is hard. Can I skip it?
+
+You are not alone. Many instructors struggle with this. If you are stuck, come back to this question after you've written the rest of the spec.
+
```

Comment:
It would be _very_ nice to stress that they **cannot** skip it though. I _always_ look at this part of the specs because if they don't figure this out while specing, they will simply exclude it during development. This may be hard, but if they cannot figure out how to distill things down to layman's terms, they might not be the right person to teach the content.

Reply:
fixed